xchange-core/src/main/java/org/knowm/xchange/dto/account/FundingRecord.java

```diff
@@ -298,7 +298,13 @@ public String toString() {
      * outflows defined above or their nature could not have been deduced from the exchanges
      * response
      */
-    OTHER_OUTFLOW(false);
+    OTHER_OUTFLOW(false),
+
+    /** Used for inflows that come from a sub account to the master account */
+    SUBACCOUNT_INFLOW(true),
```

Comment:
I think more generic names would be better as this is to Binance-specific

some exchanges have a flat structure with many account for a given currency and no master

I would suggest:

```
    /**
     * Withdrawal from the exchange a blockchain address or a bank account.
     *
     * Usually results in a withdrawal fee being subtracted
     */
    WITHDRAWAL(false),

    /** Deposit from a blockchain address or a bank account to the exchange */
    DEPOSIT(true),

    /** Used for transfers between exchanges accounts */
    INTERNAL_WITHDRAWAL(false);

    /** Used for transfers between exchanges accounts */
    INTERNAL_DEPOSIT(true),

    /** Used for transfers between exchanges accounts */
    INTERNAL_WITHDRAWAL(false);
```